docs/changelog.rst

```diff
@@ -14,7 +14,7 @@ Changelog
 
 * Python 3 support
 * Dropped support for Python 2.6
-* Added `--no-reset-stats` option for controling if the statistics should be reset once 
+* Added :code:`--no-reset-stats` option for controling if the statistics should be reset once 
```

Comment:
I think it might be worth trying to update this more often... I like this and it's a pain having to try to figure out everything that went into a change log.